static/js/biomedical/disease/chart.ts

```diff
@@ -247,3 +256,78 @@ export function drawDiseaseSymptomAssociationChart(
       (d) => `Symptom: ${d.name}<br>Odds Ratio Association: ${d.oddsRatio}`
     );
 }
+/**
+ * Draws the disease ontology hierarchy chart for the disease of interest
+ * @param id the div id where the chart is rendered on the page
+ * @param data the disease data passed into the function
+ */
+export function drawDiseaseOntologyHierarchy(
+  id: string,
+  data: DiseaseParentTree
+): void {
+  const height = GRAPH_HEIGHT - MARGIN.top - MARGIN.bottom;
+  const width = GRAPH_WIDTH - MARGIN.left;
+  // tree root
+  const root = d3.hierarchy(data).sort((a, b) => b.height - a.height);
+  // tree layout
+  const treeLayout = d3.tree().size([width, height]);
+  // calling the tree layout function for the rearrangement of root
+  treeLayout(root);
+
+  const svg = d3
+    .select(`#${id}`)
+    .append("svg")
+    .attr("width", width)
+    .attr("height", height)
+    .append("g")
+    .attr("transform", "translate(" + MARGIN.top + ")");
+
+  svg
+    .selectAll(".link")
+    .data(root.links())
+    .enter()
+    .append("line")
+    .attr("x1", function (d: any) {
+      return d.source.x;
+    })
+    .attr("y1", function (d: any) {
+      return d.source.y + NODE_VERTICAL_SHIFT;
+    })
+    .attr("x2", function (d: any) {
+      return d.target.x;
+    })
+    .attr("y2", function (d: any) {
+      return d.target.y + 3 * NODE_VERTICAL_SHIFT;
```

Comment:
why are we multiplying by 3? Maybe add a comment to explain

Reply:
Added a constant = 3, which explains it!